modules/imaging_uploader/test/imaging_uploaderTest.php

```diff
@@ -1,291 +1,12 @@
-        );
-          $this->DB->insert(
-            "candidate",
-            array(
-             'CandID'        => '999999',
-             'CenterID'      => '55',
-             'UserID'        => '1',
-             'PSCID'         => '8889',
-             'ProjectID'     => '7777'
-            )
-        );
-        $this->DB->insert(
-            "session",
-            array(
-             'ID'            => '111112',
-             'CandID'        => '999999',
-             'CenterID'      => '55',
-             'UserID'        => '1',
-             'MRIQCStatus'   => 'Pass',
-             'SubprojectID'  => '6666',
-             'Visit'         => 'In Progress',
-             'Visit_label'   => 'TestVisitLabel'
-            )
-        );
-        $this->DB->insert(
-            "mri_upload",
-            array(
-             'UploadID'                    => '9999999',
-             'UploadedBy'                  => 'test',
-             'UploadDate'                  => '2017-01-01',
-             'UploadLocation'              => '/data/incoming/test',
-             'DecompressedLocation'        => '/tmp/ImagingUpload-test',
-             'InsertionComplete'           => '0',
-             'Inserting'                   => '0',
-             'PatientName'                 => 'TestTestTest',
-             'number_of_mincInserted'      => '1',
-             'number_of_mincCreated'       => '1',
-             'TarchiveID'                  => '999999',
-             'SessionID'                   => '111112',
-             'IsCandidateInfoValidated'    => '1',
-             'IsTarchiveValidated'         => '0',
-             'IsPhantom'                   => 'N',
-            )
-        );
-    }
-    /**
-     * Deleting the test data
-     *
-     * @return void
-     */
-    function tearDown()
-    {
-        parent::tearDown();
-        $this->DB->delete(
-            "mri_upload", 
-            array('PatientName' => 'TestTestTest')
-        );
-        $this->DB->delete(
-            "session",
-            array('CandID' => '999999')
-        );
-        $this->DB->delete(
-            "candidate",
-            array('CandID' => '999999')
-        );
-        $this->DB->delete(
-            "psc",
-            array('CenterID'=>'55')
-        );
-        
-    }
-    /**
-     * Tests that, when loading the Imaging_uploader module, some
-     * text appears in the body.
-     *
-     * @return void
-     */ 
     function testImagingUploaderDoespageLoad()
     {
         $this->safeGet($this->url . '/imaging_uploader/');
         $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
         $this->assertContains("Imaging Upload", $bodyText);
     }
-    /**
-     * Tests that, when loading the Imaging_uploader module without permission,
-     * "You do not have access to this page." appears in the body.
-     *
-     * @return void
-     */
-    function testImagingUploaderLoadWithoutPermission()
```

Comment:
Hm, I can't recall why I got rid of most of them. I see some relate to the old Loris filters, but not all of them. I'll look into that